include/targets/cuda/thrust_target.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+#include <thrust/system/cuda/vector.h>
+#include <thrust/system/cuda/execution_policy.h>
```

Comment:
Thrust is system specific, so I put it into include/targets/

Reply:
Please revert this.  Thrust will be expunged from GK by the end of this week, and this will just end up causing conflicts

Reply:
OK, will do.

Reply:
Reverted in 6810f7a28 -- to be pushed